server/routes/bookRoute.js

```diff
@@ -8,5 +8,6 @@ bookRoute.post('/', Validate.addBook, BookController.addBook);
 bookRoute.get('/', BookController.getAllBooks);
 bookRoute.get('/:id', Validate.id, BookController.getSpecificBook);
 bookRoute.delete('/:id', Validate.id, BookController.deleteBook);
+bookRoute.post('/request',  Validate.requestBook, BookController.requestBook)
```

Comment:
Is this route open to the public or only authenticated users?

Reply:
It has no authentication yet.

Reply:
I think anyone should be able to request a book. Because it help the company know what books people are looking for.

Reply:
Read the story description again

Reply:
The story specified "authenticated user"